src/api/ParticipantManager.API/Data/DesignTimeDbContextFactory.cs

```diff
@@ -9,14 +9,16 @@ public class DesignTimeDbContextFactory : IDesignTimeDbContextFactory<Participan
     public ParticipantManagerDbContext CreateDbContext(string[] args)
     {
         // Build configuration to access appsettings.json or environment variables
-        var configuration = new ConfigurationBuilder()
+        new ConfigurationBuilder()
```

Comment:
We can remove this unused configuration-building code altogether. 
It doesn't have any side effects and we're not using the returned IConfigurationRoot